code/Controllers/SilverStripeNavigatorItem_Unversioned.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace SilverStripe\CMS\Controllers;
+
+use Config;
```

Comment:
I don't believe this is correct, possibly should be `use SilverStripe\Core\Config\Config;` ?

Reply:
Yes! Oops.